gb2260/division.py

```diff
@@ -119,10 +119,8 @@ def stack(self):
 def make_year_key(year):
     """A key generator for sorting years."""
     if year is None:
-        return (LATEST_YEAR, 12)
+        return (LATEST_YEAR, 7)
     year = str(year)
-    if len(year) == 4:
-        return (int(year), 12)
```

Comment:
All filenames of new source data are like "yyyyMM". So,there is not need to turn 'yyyy' to (yyyy,mm). Maybe, we should change the vairable name of year